README.md

```diff
@@ -48,6 +48,9 @@ Arangopipe has two components:
 **ArangopipeAdmin** is an administrative component. It is meant to provision projects and users into **Arangopipe**. When projects and users have been provisioned in **Arangopipe**, they can start using **Arangopipe** to track data from their machine learning experiments. To begin with, data scientists can *register* entities like datasets, featuresets and model meta-data with **Arangopipe**. Registration yields an identifier for the entity that they can use to reference the entity in their subsequent interaction with **Arangopipe**. Information provided during registration includes a component name that they can use to *lookup* the identifier for the entity using the lookup API.
 When data scientists have refined their models to a point where they are ready to track it and log its performance during model development, they can do so with a simple API call. If the model is deployment ready, they can indicate this by adding a deployment tag as part of the data provided to the model tracking API.  When models have been deployed, **Arangopipe** administrators provision a *deployment* entity in **Arangopipe** to start tracking the serving performance of the deployed model. As serving performance becomes available, it can be recorded against this deployed entity.
 
+## Picking your Arangopipe Installation
+If you need a fully functional installation with minimal configuration, follow the guidelines provided in the section 'Getting Started' (see below). If you are planning to use **Arangopipe** with a a managed services **ArangoDB** installation you will need to install the `aisis.zip` file in the `aisis-foxx` directory into your managed services **ArangoDB** installation. See the google colab notebook `arangopipe_managed_service.ipynb` in the project github repository for the details of connecting to a managed services instance of **ArangoDB**. See the instructions in the section 'Installing Arangopipe API' for the details of using the **Arangopipe** API with your pipelines or applications (this just requires the installation of the required packages).  
```

Comment:
Could we add links to sections, files, and notebooks?

Reply:
@joerg84 , will add these in.